### PR TITLE
rename master branch to main

### PR DESCRIPTION
**Pull request by nwt**

### CODE REVIEW

**Review by jameskerr — APPROVED** [review]
♥️

File: docs/Query-Library-Transfer.md [hunk 1]
```diff
@@ -100,8 +100,7 @@ $ cat appState.json | jq '.data.globalState.queries'
 # Extracting/Replacing the Query Library
 
 For ease of use, we've created a simple Python script
-[`qlib_util.py`](https://raw.githubusercontent.com/brimdata/brim/master/scripts/util/qlib_util.py)
-that can easily perform this.
+[`qlib_util.py`](../scripts/util/qlib_util.py) that can easily perform this.
```
Comment (philrz) [inline docs/Query-Library-Transfer.md:103]:
```suggestion
[`qlib_util.py`](https://raw.githubusercontent.com/brimdata/brim/main/scripts/util/qlib_util.py) that can easily perform this.
```

The original change was going to break the [wiki](https://github.com/brimdata/brim/wiki). When I set up the wiki, there was a strong desire in the team to still have the original Markdown docs live in the main code repo, so we have a [docs-publish.yml](https://github.com/brimdata/brim/blob/main/.github/workflows/docs-publish.yml) Actions Workflow that copies those Markdown files from `docs/` into the appropriate spots in the separate meta-repo that holds the actual wiki. Links in that wiki repo are not relative to the main repo, so I'm pretty sure we need to maintain the original absolute link with `main` in it. Sorry if that's disappointing.

File: docs/Supported-Platforms.md [hunk 2]
```diff
@@ -140,11 +140,11 @@ The most extensive testing of Brim is provided via automation that is run on
 versions of
 [Runners](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners)
 are referenced in the automation for Brim's
-[Continuous Integration tests](https://github.com/brimdata/brim/blob/master/.github/workflows/ci.yml)
+[Continuous Integration tests](../.github/workflows/ci.yml)
```
Comment (philrz) [inline docs/Supported-Platforms.md:143]:
```suggestion
[Continuous Integration tests](https://github.com/brimdata/brim/blob/main/.github/workflows/ci.yml)
```

File: docs/Supported-Platforms.md [hunk 3]
```diff
@@ -140,11 +140,11 @@ The most extensive testing of Brim is provided via automation that is run on
...
 and build workflows for
-[Windows](https://github.com/brimdata/brim/blob/master/.github/workflows/win-release-candidate.yml),
-[macOS](https://github.com/brimdata/brim/blob/master/.github/workflows/macos-release-candidate.yml), and
-[Linux](https://github.com/brimdata/brim/blob/master/.github/workflows/linux-release-candidate.yml).
+[Windows](../.github/workflows/win-release-candidate.yml),
+[macOS](../.github/workflows/macos-release-candidate.yml), and
+[Linux](../.github/workflows/linux-release-candidate.yml).
```
Comment (philrz) [inline docs/Supported-Platforms.md:147]:
```suggestion
[Windows](https://github.com/brimdata/brim/blob/main/.github/workflows/win-release-candidate.yml),
[macOS](https://github.com/brimdata/brim/blob/main/.github/workflows/macos-release-candidate.yml), and
[Linux](https://github.com/brimdata/brim/blob/main/.github/workflows/linux-release-candidate.yml).
```

**Review by philrz — CHANGES_REQUESTED** [review]
I think this needs to revert a couple things to avoid breaking the Brim wiki.

File: docs/Query-Library-Transfer.md (hunk 1 above)
Comment (nwt) [reply docs/Query-Library-Transfer.md]: Thanks for pointing this out!

25388b4 restores absolute under `docs/`.

**Review by philrz — APPROVED** [review]

**Pull request merged**